packages/p2p/src/components/app.jsx

```diff
@@ -16,6 +17,13 @@ import Routes from './routes/routes.jsx';
 import './app.scss';
 
 const App = () => {
+    const list_item_limit = is_mobile ? 10 : 50;
+    const { data, error } = useP2POrderList({
+        p2p_order_list: 1,
+        subscribe: 1,
+        offset: 0,
+        limit: list_item_limit,
+    });
```

Comment:
```suggestion
    const { data, error } = useP2POrderList();
```
@yaswanth-deriv It doesn't even accept `p2p_order_list` and `subscribe` as parameters 🤔 Anyway, you don't need to pass anything here 🤔

Reply:
Yeah I will remove that..Thanks